documentation/1.0.0/dockerized/docker-compose-project.yml

```diff
@@ -43,6 +43,7 @@ services:
 
   project:
     image: shikshalokamqa/elevate-project-service:1.0.0
+    container_name: project
```

Comment:
@Prajwal17Tunerlabs will this change break anything for linux, mac setup ?
cc- @MallanagoudaB

Reply:
no @VISHNUDAS-tunerlabs